dropshot/src/handler.rs

```diff
@@ -1001,7 +1001,7 @@ where
  * `UntypedBody` is an extractor for reading in the contents of the HTTP request
  * body and making the raw bytes directly available to the consumer.
  */
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
It seems like no problem to put this onto `UntypedBody`.

I'm less sure about `TypedBody<BodyType>` only because I think it means that the caller has to impl `Clone` on their `BodyType`.  I'd be interested in what others think about that.